mmv1/third_party/terraform/website/docs/r/gke_hub_feature.html.markdown

```diff
@@ -71,6 +71,66 @@ resource "google_gke_hub_feature" "feature" {
 }
 ```
 
+## Example Usage - Enable Fleet Observability for default logs with COPY
+
+```hcl
+resource "google_gke_hub_feature" "feature" {
+  name = "fleetobservability"
+  location = "global"
+  spec {
+    fleetobservability {
+      logging_config {
+        default_config {
+          mode = "COPY"
+        }
+      }
+    }
+  }
+  provider = google-beta
+}
+```
```

Comment:
Any examples to unset the feature?

Reply:
Resolve to move our discussion to b/286127157. Will update the doc once we have the update.